huaweicloud/services/acceptance/dew/resource_huaweicloud_kms_key_test.go

```diff
@@ -270,6 +333,18 @@ resource "huaweicloud_kms_key" "key_1" {
 `, keyAliasUpdate)
 }
 
+func testAccKmsImportKeyUpdate(keyAliasUpdate string) string {
```

Comment:
this function just update key_alias field, It has already been done in other functions, this can be delete